autofit/non_linear/nest/dynesty/abstract.py

```diff
@@ -436,11 +436,11 @@ def should_plot(name):
         if should_plot("cornerplot"):
             plotter.cornerplot()
 
-        if should_plot("runplot"):
-            plotter.runplot()
+        # if should_plot("runplot"):
+        #     plotter.runplot()
 
         if should_plot("traceplot"):
             plotter.traceplot()
 
-        if should_plot("cornerpoints"):
-            plotter.cornerpoints()
+        # if should_plot("cornerpoints"):
+        #     plotter.cornerpoints()
```

Comment:
?

Reply:
The combination of them is causing a visualization bug, where the internal dynesty plots are messing up the matplootlib figure of subsequent plots.

I want to keep the commented code as my hope is that it will be fixed in a new dynesty release, but I'll put in a comment explaining this.